### PR TITLE
Initial implementation for Cancel ruling

### PR DESCRIPTION
**Pull request by SteveSugden**
No Unit/Integration tests

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala [hunk 1]
```diff
@@ -60,6 +60,14 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     )
   }
 
+  def auditRulingCancelled(oldCase: Case, updatedCase: Case, operator: Operator)
+                       (implicit hc: HeaderCarrier): Unit = {
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: formatting

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala [hunk 2]
```diff
@@ -60,6 +60,14 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
     )
   }
 
+  def auditRulingCancelled(oldCase: Case, updatedCase: Case, operator: Operator)
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: cancel case or cancel ruling?

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala [hunk 3]
```diff
@@ -83,4 +91,5 @@ object AuditPayloadType {
   val CaseReferred = "caseReferred"
   val CaseReleased = "caseReleased"
   val CaseCompleted = "caseCompleted"
+  val RulingCancelled = "rulingCancelled"
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: this needs to be reviewed - are there other actions for cancelling cases? or is your action the only cancel action?

**Comment by gokyo** [comment]
@SteveSugden 
unit and integration tests are missing

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala (hunk 2 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: I believe its canceling the ruling - at least in the ticket and the prototype UI.  I've tried to be consistent with this in the code.

File: app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala (hunk 3 above)
Comment (SteveSugden) [reply app/uk/gov/hmrc/tariffclassificationfrontend/audit/AuditService.scala]: I don't think so but I will check with Joe.

**Comment by SteveSugden** [comment]
I know there are no tests @gokyo - this was a "preview".  I'm going to close this PR and carry on working on the tests

**Pull request closed without merging**